CMakeLists.txt

```diff
@@ -534,6 +534,10 @@ if (BUILD_TOOLS)
     add_definitions(-DWITH_WASI)
     set(INTERP_LIBS uvwasi_a)
     set(EXTRA_INTERP_SRC src/interp/interp-wasi.cc)
+
+    if (COMPILER_IS_GNU)
```

Comment:
Is this macro also defined when clang is the compiler?

Reply:
Yes, it looks like we always set it to either 0 or 1 (here: https://github.com/WebAssembly/wabt/blob/main/CMakeLists.txt#L73-L93) and then use it in a bunch of places.